Add scripts that implement a cron job that syncs logs to a server

mgbailey opened the pull request:
More details are in the README file.

Commit:
Add scripts that implement a cron job that syncs logs to a server

More details are in the README file.

absltkaos commented on log_aggregation/install_logger.sh:
```diff
@@ -0,0 +1,17 @@
+ssh-keygen -q -f ./logwriter_key -t ed25519 -N ''
```
Probably should have a shebang on this.

mgbailey replied:
Yeah, I will add that just in case someone is running from an oddball shell. Thanks

Commit:
Minor refactoring. Added instructions to README for repo download

Refactored to eliminate code duplication. Made use of single- vs.
double-quotes more consistant. Modified README to give
instructions for cloning github repo instead of uppacking a tar
file archive.

mgbailey commented:
Quashing this, and will resubmit with recommended changes.
